python/cudf_polars/cudf_polars/dsl/expressions/unary.py

```diff
@@ -339,6 +341,21 @@ def do_evaluate(
                 ),
                 dtype=self.dtype,
             )
+        elif self.name == "top_k":
+            (column, k) = (
+                child.evaluate(df, context=context) for child in self.children
+            )
+            (reverse,) = self.options
+            return Column(
+                plc.sorting.top_k(
+                    column.obj,
+                    k.obj.to_scalar().to_py(),
```

Comment:
Is `k` a `Literal` expression before evaluation? If so can we just use `self.children[0].value` to avoid to conversion to GPU?